Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM centos:7
+ENV container docker
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
+systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+rm -f /lib/systemd/system/multi-user.target.wants/*;\
+rm -f /etc/systemd/system/*.wants/*;\
+rm -f /lib/systemd/system/local-fs.target.wants/*; \
+rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+rm -f /lib/systemd/system/basic.target.wants/*;\
+rm -f /lib/systemd/system/anaconda.target.wants/*;
+VOLUME [ "/sys/fs/cgroup" ]
+
+# HTCondor User details:
+ENV SUBMIT_USER submitter
+ENV GID 1000
+ENV UID 1000
+ENV PASS password
+
+# Build in one RUN
+RUN yum -y install \
+         yum-utils \
+         sudo \
+         which \
+         openssh-clients && \
+    yum -y groupinstall 'Development Tools' 
+
+RUN curl -L -s http://research.cs.wisc.edu/htcondor/yum/RPM-GPG-KEY-HTCondor > RPM-GPG-KEY-HTCondor && \
+    rpm --import RPM-GPG-KEY-HTCondor && \
+    yum-config-manager --add-repo https://research.cs.wisc.edu/htcondor/yum/repo.d/htcondor-development-rhel7.repo && \
+    yum clean all && \
+    rm -f RPM-GPG-KEY-HTCondor && \
+    yum -y install condor && \
+    groupadd -g ${GID} ${SUBMIT_USER} && \
+    useradd -m -u ${UID} -g ${GID} ${SUBMIT_USER} && \
+    usermod -a -G condor ${SUBMIT_USER} && \
+    echo ${PASS} | passwd --stdin ${SUBMIT_USER} && \
+    sed -i 's/\(^Defaults.*requiretty.*\)/#\1/' /etc/sudoers && \
+    systemctl enable condor
+
+RUN yum -y install epel-release httpd mod_wsgi mod_ssl net-tools vim
+RUN systemctl enable httpd
+
+COPY docker/hostkey.pem /etc/pki/tls/private/localhost.key
+COPY docker/hostcert.pem /etc/pki/tls/certs/localhost.crt
+COPY docker/igtf-ca-bundle.crt /etc/pki/tls/certs/server-chain.crt
+RUN mv /etc/httpd/conf.d/ssl.conf /etc/httpd/conf.d/ssl.conf.orig && \
+    sed 's/#SSLCertificateChainFile/SSLCertificateChainFile/g' /etc/httpd/conf.d/ssl.conf.orig > /etc/httpd/conf.d/ssl.conf && \
+    rm -f /etc/httpd/conf.d/ssl.conf.orig
+
+RUN yum -y install python2-pip
+RUN pip install --upgrade pip
+RUN pip install git+https://github.com/htcondor/scitokens-credmon
+
+# KNOBS
+COPY docker/10-docker.conf /etc/condor/config.d/
+RUN echo "CREDD_OAUTH_MODE = TRUE" > /etc/condor/config.d/70-startd-token.conf
+RUN echo "SEC_DEFAULT_ENCRYPTION = REQUIRED" > /etc/condor/config.d/10-sec-encrypt.conf
+COPY docker/60-oauth-token-providers.conf.tmpl /etc/condor/config.d/60-oauth-token-providers.conf.tmpl
+
+RUN scitokens_credmon --deploy --apache
```

Comment:
This is on me for changing the deployment so much, but this isn't the right thing to do anymore. I don't have them yet, but I'll figure out and request changes soon. Will need a way to pull the example configs... maybe we have the Docker image grab a release tarball and install the credmon from it instead of doing a `pip install` from PyPI.